install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -77,74 +69,97 @@ Because the Aggregator PV is relatively small, the least expensive performance g
+  # image provided (registry, image, tag) will be used for aggregator.
+  # fullImageName:
+  imagePullPolicy: IfNotPresent
+
+  # For legacy configuration support, `enabled: true` overrides deployMethod
+  # and causes `deployMethod: "statefulset"`
+  enabled: false
+
+  # Replicas sets the number of Aggregator replicas. It only has an effect if
+  # `deployMethod: "statefulset"`
+  replicas: 1
+
 
-    # How much data to ingest from the federated store bucket, and how much data
-    # to keep in the DB before rolling the data off.
-    # 
-    # Note: If increasing this value to backfill historical data, it will take
-    # time to gradually ingest & process those historical ETL files. Consider
-    # also increasing the resources available to the aggregator as well as the
-    # refresh & concurrency env vars.
-    # 
-    # default: 91
-    ETL_DAILY_STORE_DURATION_DAYS: "365"
-    
-    # How many threads to use when ingesting Asset/Allocation/CloudCost data
-    # from the federated store bucket. In most cases the default is sufficient,
-    # but can be increased if trying to backfill historical data.
-    # default: 3
-    DB_CONCURRENT_INGESTION_COUNT: "5"
-
-    # Memory limit applied to read database connections.
-    # default: n/a
-    DB_MEMORY_LIMIT: 8GB
-
-    # Memory limit applied to write database connections.
-    # default: n/a
-    DB_WRITE_MEMORY_LMIT: 8GB
-
-    # How many threads the read database is configured with (i.e. Kubecost API /
-    # UI queries). If increasing this value, it is recommended to increase the
-    # aggregator's memory requests & limits.
-    # default: 1
-    DB_READ_THREADS: 3
-
-    # How many threads the write database is configured with (i.e. ingestion of
-    # new data from S3). If increasing this value, it is recommended to increase
-    # the aggregator's memory requests & limits.
-    # default: 1
-    DB_WRITE_THREADS: 3
-
-    # log level
-    # default: info
-    LOG_LEVEL: info
+  # stagingEmptyDirSizeLimit changes how large the "staging"
+  # /var/configs/waterfowl emptyDir is. It only takes effect in StatefulSet
+  # configurations of Aggregator, other configurations are unaffected.
+  #
+  # It should be set to approximately 8x the size of the largest bingen file in
+  # object storage. For example, if your largest bingen file is a daily
+  # Allocation file with size 300MiB, this value should be set to approximately
+  # 2400Mi. In most environments, the default should suffice.
+  stagingEmptyDirSizeLimit: 2Gi
+
+  # this is the number of partitions the datastore is split into for copying
+  # the higher this number, the lower the ram usage but the longer it takes for
+  # new data to show in the kubecost UI
+  # set to 0 for max partitioning (minimum possible ram usage, but the slowest)
+  # the default of 25 is sufficient for 95%+ of users. This should only be modified
+  # after consulting with Kubecost's support team
```

Comment:
@jessegoodier I think this comment may no longer be relevant? I think I remember you found that `0` won't work for this config. And from what I remember setting it to `1` will allow for a faster copy, but more memory usage.

Reply:
I'd like to get a full eng review on tuning and recommendations. 
agree the statefulSet should set here and remove defaults that will rarely, if ever be modified.

Reply:
Yep sounds good. I will propose a new PR sometime soon and let's get a full eng review on it.